From 741e61dbd8a12f25fdab237edcfc61aa4910cc64 Mon Sep 17 00:00:00 2001
From: Manish Regmi <manish.regmi@intel.com>
Date: Wed, 6 Aug 2014 14:40:12 -0700
Subject: [PATCH] GMIN-2603: pnpdriver does not recognize some resources

PNP driver currently does not recognise GPIO and SERIAL bus resource
and give out unknown resource errors in the kernel log.
We just add these two resource types in the switch statement.

Bug: GMIN-2603
Change-Id: Ic17e16e347faa5be1bd291ccad2d3d620897882e
Signed-off-by: Manish Regmi <manish.regmi@intel.com>
(cherry picked from commit 7816878ae5ec0acda56b12668dd0a5ef767343a3)

Signed-off-by: Brett T. Warden <brett.t.warden@intel.com>
---
 drivers/pnp/pnpacpi/rsparser.c |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/drivers/pnp/pnpacpi/rsparser.c b/drivers/pnp/pnpacpi/rsparser.c
index 66977eb..9306940 100644
--- a/drivers/pnp/pnpacpi/rsparser.c
+++ b/drivers/pnp/pnpacpi/rsparser.c
@@ -251,6 +251,12 @@ static acpi_status pnpacpi_allocated_resource(struct acpi_resource *res,
 	case ACPI_RESOURCE_TYPE_GENERIC_REGISTER:
 		break;
 
+	case ACPI_RESOURCE_TYPE_GPIO:
+		break;
+
+	case ACPI_RESOURCE_TYPE_SERIAL_BUS:
+		break;
+
 	default:
 		dev_warn(&dev->dev, "unknown resource type %d in _CRS\n",
 			 res->type);
-- 
1.7.9.5

